pymobiledevice3/__main__.py

```diff
@@ -92,6 +93,8 @@ def cli():
         logger.error('Unable to launch application (try to unlock device)')
     except WebInspectorNotEnabledError:
         logger.error('Web inspector is not enable')
+    except RemoteAutomationNotEnabled:
```

Comment:
The main try except handles more low level exception, you should it put it under webinspector